contracts/test/utils.ts

```diff
@@ -50,9 +55,42 @@ export async function setNextBlockTimestamp(timestamp: BigNumberish): Promise<nu
   return timestamp;
 }
 
+// --- Uniswap SDK Helpers ---
+export async function encodeRoute(tokens: SupportedToken[]): Promise<string> {
```

Comment:
@apbendi / @mds1 would it make sense to move this into it's utils.ts file which is not under `test` cause this would be invoked by dApp as well right ?

Reply:
It would definitely be nice to share such utilities across app and contract tests. Right now, we do share [types](https://github.com/dcgtc/dgrants/tree/main/types) between them, but I think that's it. I wonder how difficult it would be to pull these helpers into a shared method. Any thoughts here @wildmolasses.

Reply:
I agree we want this eventually, but I don't think we have an issue/spec for integrating swaps into the app so it also may not be worth delaying this PR to share these with the frontend, especially since this version of the functions is too probably simple for the app